core/controllers/base.py

```diff
@@ -387,7 +390,7 @@ def render_template(
         self.response.pragma = 'no-cache'
 
         self.response.write(self.jinja2_env.get_template(
-            filename).render(**values))
+            'pages/%s' % filename).render(**values))
```

Comment:
Fair point, I wasn't sure either. Shall I just revert it and have every handler call pages/*?